shell/platform/android/test/io/flutter/plugin/editing/TextInputPluginTest.java

```diff
@@ -273,7 +273,7 @@ public void textEditingDelta_TestUpdateEditingValueWithDeltasIsNotInvokedWhenDel
             true,
             false, // Delta model is disabled.
             TextInputChannel.TextCapitalization.NONE,
-            new TextInputChannel.InputType(TextInputChannel.TextInputType.TEXT, false, false),
+            new TextInputChannel.InputType(TextInputChannel.TextInputType.MULTILINE, false, false),
```

Comment:
Why this change?

Reply:
Oops my bad... that's the test I copy/pasted to bootstrap the new one, it looks like I updated it by mistake at one point. :facepalm: